webapp/src/ts/services/enketo.service.ts

```diff
@@ -713,6 +714,21 @@ export class EnketoService {
       });
   }
 
+  private validateAttachments(docs) {
```

Comment:
Hi @dianabarsan, only if you have time, can I have an early opinion about this approach? 

It's not complete, I need to add test coverage, translation key and ask Nicole if we want to display the `The uploaded files exceed total size limit.` in the snack bar or in the form 🤔

Reply:
This is a good idea. 
I think the calculation can be changed so it validates the maximum size per document, not per group of documents. With replication batch size going down to 1, we only need to make sure every document individually can be uploaded. 
I'm most worried about PouchDb crashing completely with large attachments, which would render the device unusable.

Reply:
Done